Makefile

```diff
@@ -1,21 +1,21 @@
 .PHONY: docs
 init:
-	pip install pipenv --upgrade
-	pipenv install --dev
+	pip install -e .[socks]
+	pip install -r requirements-dev.txt
 test:
 	# This runs all of the tests, on both Python 2 and Python 3.
 	detox
 ci:
-	pipenv run py.test -n 8 --boxed --junitxml=report.xml
+	pytest tests --junitxml=report.xml
```

Comment:
Since we're (rightfully) dropping pipenv, should we get rid of the rest of its cruft (Pipenv, Pipenv.lock)?

Reply:
I'd been thinking we'd do that in the next minor release. Pipenv has ~1M downloads a month, but I have no idea if anyone even uses the Requests' Pipfile. I'm not tied to waiting if you or Seth feel strongly.